src/stock-operations/batch-no-selector/batch-no-selector.component.tsx

```diff
@@ -78,36 +71,26 @@ const BatchNoSelector = <T,>(props: BatchNoSelectorProps<T>) => {
 
   return (
     <div style={{ display: "flex", flexDirection: "column" }}>
-      <Controller
-        name={props.controllerName}
-        control={props.control}
-        render={({ field: { onChange, ref } }) => (
-          <ComboBox
-            style={{ flexGrow: "1" }}
-            titleText={props.title}
-            name={props.name}
-            control={props.control}
-            controllerName={props.controllerName}
-            id={props.name}
-            size={"sm"}
-            items={stockItemBatchesInfo || []}
-            onChange={(data: { selectedItem?: StockBatchDTO }) => {
-              setSelectedItem(data.selectedItem || null);
-              props.onBatchNoChanged?.(data.selectedItem);
-              onChange(data.selectedItem?.uuid);
-            }}
-            initialSelectedItem={initialSelectedItem}
-            itemToString={(s: StockBatchDTO) =>
-              s?.batchNo ? `${s?.batchNo} | Qty: ${s?.quantity ?? ""}` : ""
-            }
-            placeholder={props.placeholder}
-            invalid={props.invalid}
-            invalidText={props.invalidText}
-            ref={ref}
-          />
-        )}
+      <ComboBox
```

Comment:
is there any particular reason this needs to change from a controlledDropDown to just a combox ??

Reply:
@ODORA0 without controller, the field won't be validated or recorded by the zod implementation, please check that. Also this is a reusable component, we should be careful modifying it.